capplets/common/mate-theme-apply.c

```diff
@@ -68,6 +85,11 @@ mate_meta_theme_set (MateThemeMetaInfo *meta_theme_info)
   if (compare (old_key, meta_theme_info->gtk_theme_name))
     {
       g_settings_set_string (interface_settings, GTK_THEME_KEY, meta_theme_info->gtk_theme_name);
+
+      if (wayland == TRUE)
```

Comment:
Actually I'd have done it the *other* way around and only checked whether the gsettings was there, assuming whatever the check was for setting it was right:
```suggestion
      if (interface_gnome_settings)
```

Reply:
Interesting, that simplifies the code a bit, will do

Reply:
done